test/integration/tests/pcrextend.sh

```diff
@@ -13,6 +13,20 @@ declare -A alg_hashes=(
 )
 
 digests=""
+## Check cpHash output for TPM2_PCR_Extend
+tpm2_pcrextend 4:sha1=f1d2d2f924e986ac86fdf7b36c94bcdf32beec15 --cphash cp.hash
+TPM2_CC_PCR_Extend="00000182"
+pcrHandle="00000000"
```

Comment:
```suggestion
#pcrHandle="00000000"
```

Reply:
Not sure why pcrhandle and size needs to be dropped - but doing so evaluates the expected cpHash value. Looks like an issue in SAPI or getcphash routine in the tools library.